models/AmazonS3.cfc

```diff
@@ -351,35 +351,62 @@ component accessors="true" singleton {
 		);
 
 		return results.responseheader.status_code == 200;
+    }
+
+    /**
+     * Gets a bucket's or object's ACL policy.
+     *
+     * @bucketName The bucket to get the ACL.
+     * @uri        An optional resource uri to get the ACL.
+     *
+     * @return     An array containing the ACL for the given resource.
+     */
+    array function getAccessControlPolicy( required string bucketName=variables.defaultBucketName, string uri = "" ) {
+        requireBucketName( arguments.bucketName );
+        var resource = arguments.bucketName;
+
+		if ( len( arguments.uri ) ) {
+            resource = resource & "/" & arguments.uri;
+        }
+
+        var results = S3Request( resource = resource, parameters={ "acl" : true } );
+
+        var grantsXML = xmlSearch( results.response, "//*[local-name()='Grant']" );
+        return arrayMap( grantsXML, function( node ) {
+            return {
+                "type" 		  = node.grantee.XMLAttributes[ "xsi:type" ],
+                "displayName" = "",
+                "permission"  = node.permission.XMLText,
+                "uri"         = node.grantee.XMLAttributes[ "xsi:type" ] == "Group" ? node.grantee.uri.xmlText : node.grantee.displayName.xmlText
+            };
+        } );
 	}
 
+
 	/**
-	 * Gets a bucket's or object's ACL policy.
-	 *
-	 * @bucketName The bucket to get the ACL.
-	 * @uri        An optional resource uri to get the ACL.
-	 *
-	 * @return     An array containing the ACL for the given resource.
-	 */
-	array function getAccessControlPolicy( required string bucketName = variables.defaultBucketName, string uri = "" ) {
+     * Sets a bucket's or object's ACL policy.
+     *
+     * @bucketName The bucket to get the ACL.
+     * @uri        An optional resource uri to get the ACL.
+	 * @acl        A known ACL string
+     *
+     */
```

Comment:
Would be great to add a `@see https://link.to.amzon/acl/policies` here. So we can have that on the generated docs.